src/gcodeExport.cpp

```diff
@@ -197,6 +201,7 @@ std::string GCodeExport::getFileHeader(const double* print_time, const std::vect
         else if (flavor == EGCodeFlavor::REPRAP)
         {
             prefix << ";Filament used: " << ((filament_used.size() >= 1)? filament_used[0] / (1000 * extruder_attr[0].filament_area) : 0) << "m" << new_line;
+            prefix << ";Layer height: " << layer_height << new_line;
```

Comment:
Sorry if this is me being stupid but if I declare it above that line it won't have file scope will it?

I understand what you are saying about having different layer heights per meshgroup but I don't think the Duet (reprap) firmware really knows about printing multiple objects sequentially so this is the best we can do.

If we added this comment to the starting code then it would have to be changed for each different layer height used, surely it is better to generate it automatically?